index.rst

```diff
@@ -0,0 +1,29 @@
+.. ld_python_client documentation master file, created by
```

Comment:
I actually think they're not automatically generated. I had to make them using a separate command, and RTD says that it looks for a conf.py and any .rst files to build, rather than building them itself.